README.adoc

```diff
@@ -1,17 +1,4 @@
-|===
-|Plugin Information
```

Comment:
this is sourced from update centre data automatically on the plugin site, and uses much nicer looking UI there